internal.nix

```diff
@@ -67,7 +67,8 @@ rec {
             } else
           builtins.fetchGit {
             url = "https://github.com/${org}/${repo}";
-            inherit rev ref;
+            inherit rev;
+            allRefs = true;
```

Comment:
If this ia ll it takes this sounds good. Just have to make sure it doesn't break on older versions of nix (2.3)